CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+# Contribution guidelines
+
+Ongoing development on this project is managed by this project board: https://github.com/neontribe/www/projects/6.
+
+## Getting started on development
+
+Any issues in the `To do` column are assumed to be important enough and have been prioritised.
+
+When contributing to this project please do the following:
+
+1. Pick something near the top of the to do column, which you feel you have the time and confidence to tackle.
+   - We expect you to learn while tackling issues, please don't be put off by a tasks' seeming complexity.
+   - If you need more information please follow up with the person who spawned the task.
```

Comment:
```suggestion
   - If you need more information please follow up with the person who spawned/created the task.
```
I almost went for a change to plain English and got rid of spawned - but then thought actually, no its good for people to learn coders language as they start working on this - what do people feel about using both the coders slang term and the plain english term for things like this?